### PR TITLE
add integration test for alphavantage

### PR DESCRIPTION
**Pull request by karen-stepanyan**
## Changes

Add integration tests for alphavantage EA

## Steps to Test

`yarn test packages/sources/alphavantage/test/integration`

## Quality Assurance

- [ ] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `<ADAPTER_PACKAGE>/schemas/env.json` and `<ADAPTER_PACKAGE>/README.md`
- [ ] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `infra-k8s` configuration file.
- [ ] The branch naming follows git flow (`feature/x`, `chore/x`, `release/x`, `hotfix/x`, `fix/x`) or is created from Clubhouse/Shortcut
- [ ] This is related to a maximum of one Clubhouse/Shortcut story or GitHub issue
- [x] Types are safe (avoid TypeScript/TSLint features like any and disable, instead use more specific types)

### CODE REVIEW

File: .idea/workspace.xml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+      <change beforePath="$PROJECT_DIR$/.pnp.cjs" beforeDir="false" afterPath="$PROJECT_DIR$/.pnp.cjs" afterDir="false" />
+      <change beforePath="$PROJECT_DIR$/packages/sources/alphavantage/package.json" beforeDir="false" afterPath="$PROJECT_DIR$/packages/sources/alphavantage/package.json" afterDir="false" />
+      <change beforePath="$PROJECT_DIR$/packages/sources/alphavantage/src/index.ts" beforeDir="false" afterPath="$PROJECT_DIR$/packages/sources/alphavantage/src/index.ts" afterDir="false" />
+      <change beforePath="$PROJECT_DIR$/yarn.lock" beforeDir="false" afterPath="$PROJECT_DIR$/yarn.lock" afterDir="false" />
+    </list>
+    <option name="SHOW_DIALOG" value="false" />
+    <option name="HIGHLIGHT_CONFLICTS" value="true" />
+    <option name="HIGHLIGHT_NON_ACTIVE_CHANGELIST" value="false" />
+    <option name="LAST_RESOLUTION" value="IGNORE" />
+  </component>
```
Comment (austinborn) [inline .idea/workspace.xml:14]: What is this for?

Comment (karen-stepanyan) [reply .idea/workspace.xml:14]: This is the folder that is generated by my idea (webstorm). I know you all probably using vscode. I guess we either need to add this to gitignore, or just include it as well (like we did for vscode). Currently i'm manually removing this folder each time, but it gets regenerated again.

Comment (karen-stepanyan) [reply .idea/workspace.xml]: removed again and pushed, but i guess we need to solve this another way

Comment (austinborn) [reply .idea/workspace.xml]: Got it. If we follow the precedent of `vscode`, I think we should include it then

Comment (austinborn) [reply .idea/workspace.xml]: Or just update the `gitignore`. I'll leave it up to your discretion

**Review by austinborn — APPROVED** [review]

**Pull request merged**